sdk/src/test/java/com/mapbox/maps/MapControllerTest.kt

```diff
@@ -260,9 +261,12 @@ class MapControllerTest {
 
   @Test
   fun setMaximumFpsInvalid() {
+    every { mockRenderer.setMaximumFps(any()) } just Runs
+
     testMapController.setMaximumFps(-1)
 
-    verify(exactly = 0) { mockRenderer.setMaximumFps(any()) }
+    // range check is performed within MapboxRenderer
+    verify { mockRenderer.setMaximumFps(any()) }
```

Comment:
```suggestion
    verifyNo { mockedRenderThread.setUserRefreshRate(any()) }
```

Reply:
it should be verify here, I did even leave a comment